debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleDatabaseSchema.java

```diff
@@ -42,6 +43,10 @@ public OracleDatabaseSchema(OracleConnectorConfig connectorConfig, SchemaNameAdj
                 connectorConfig.getKeyMapper());
     }
 
+    public Tables getTables() {
```

Comment:
Is that needed, doesn't add any value it seems?

Reply:
It wouldn't be needed if we change `RelationalDatabaseSchema#tables()` as `public` rather than `protected`.  I wonder if it would simply make more sense to leave this for now as only Oracle needs access to the `Tables` object this way and we look at making changes to Oracle to avoid needing access to this to follow how other connectors behave instead?

Reply:
Sounds good.